components/org.wso2.carbon.sp.jobmanager.core/src/test/java/org/wso2/carbon/sp/jobmanager/core/SiddhiTopologyCreatorTestCase.java

```diff
@@ -1434,27 +1435,28 @@ public void testPassthoughWithMultipleSubscription() {
      */
     @Test(dependsOnMethods = "testPassthoughWithMultipleSubscription")
     public void testUsergivenSourceSingleGroup() {
-        String siddhiApp = "@App:name('MB-Testcase-ptTest')\n" +
-                "@App:description('Testing the MB implementation with passthrough fix.')\n"+
-                "@source(type = 'http', receiver.url='http://localhost:8080/SweetProductionEP', @map(type = 'json'))\n"+
-                "define stream Test1Stream (name string, amount double);\n"+
-                "@sink(type='log')\n"+
-                "define stream Test2Stream (name string, amount double);\n"+
-                "@info(name = 'query1')@dist(parallel='3', execGroup='001')\n"+
-                "Partition with (name of Test1Stream)\n"+
-                "Begin\n"+
-                "from Test1Stream\n"+
-                "select name,amount\n"+
-                "insert into Test2Stream;\n"+
-                "end";
+        String siddhiApp = "@App:name('MB-Testcase-ptTest')\n"
```

Comment:
Shall we change the app name and description as this test case has nothing to do with MB